core/src/main/scala/org/apache/spark/scheduler/InputFormatInfo.scala

```diff
@@ -162,6 +163,7 @@ object InputFormatInfo {
     go to (a) until required nodes are allocated.
 
     If a node 'dies', follow same procedure.
+    }}}
```

Comment:
If this is just to get around the `->`, I would instead change that to "to" and leave this. Or, if you like, make it an HTML ordered list